app/tests/test_item_to_collection_endpoint.py

```diff
@@ -0,0 +1,197 @@
+
+        response_item_west = self.client.get(
+            f"/{API_BASE}/collections/{collection_name}/items/{item_west}"
+        )
+        bbox_item_west = response_item_west.json()['bbox']
+
+        self.assertEqual(self._round_list(bbox_collection), self._round_list(bbox_item_west))
+
+    def test_update_covering_item(self):
+        collection_name = self.collection.name
+        item_name = self.items[0].model.name
+        sample = self.factory.create_item_sample(
+            self.collection, sample='item-covers-switzerland', name=item_name
+        )
+        path = f'/{API_BASE}/collections/{self.collection.name}/items/{item_name}'
+        response = self.client.put(
+            path, data=sample.get_json('put'), content_type="application/json"
+        )
+
+        response_item = self.client.get(
+            f"/{API_BASE}/collections/{collection_name}/items/{item_name}"
+        )
+        bbox_item = response_item.json()['bbox']
+
+        response_collection = self.client.get(f"/{API_BASE}/collections/{collection_name}")
+        bbox_collection = response_collection.json()['extent']['spatial']['bbox'][0]
+
+        self.assertEqual(self._round_list(bbox_collection), self._round_list(bbox_item))
+
+    def test_add_covering_item(self):
+        collection_name = self.collection.name
+        response_collection = self.client.get(f"/{API_BASE}/collections/{collection_name}")
+        bbox_collection_ch = response_collection.json()['extent']['spatial']['bbox'][0]
+
+        sample = self.factory.create_item_sample(
+            self.collection, sample='item-covers-switzerland', db_create=True
+        ).model
+        response_collection = self.client.get(f"/{API_BASE}/collections/{collection_name}")
+        bbox_collection_covers_ch = response_collection.json()['extent']['spatial']['bbox'][0]
+
+        self.assertNotEqual(
+            self._round_list(bbox_collection_ch), self._round_list(bbox_collection_covers_ch)
+        )
+
+        polygon_ch = Polygon.from_bbox(bbox_collection_ch)
+        polygon_covers_ch = Polygon.from_bbox(bbox_collection_covers_ch)
+
+        self.assertGreater(polygon_covers_ch.area, polygon_ch.area)
+
+    def test_add_another_item(self):
+        collection_name = self.collection.name
+        response_collection = self.client.get(f"/{API_BASE}/collections/{collection_name}")
+        bbox_collection_ch = response_collection.json()['extent']['spatial']['bbox'][0]
+
+        sample = self.factory.create_item_sample(
+            self.collection, sample='item-paris', db_create=True
+        ).model
+        response_collection = self.client.get(f"/{API_BASE}/collections/{collection_name}")
+        bbox_collection_paris = response_collection.json()['extent']['spatial']['bbox'][0]
+
+        self.assertNotEqual(
+            self._round_list(bbox_collection_ch), self._round_list(bbox_collection_paris)
+        )
+
+        polygon_ch = Polygon.from_bbox(bbox_collection_ch)
```

Comment:
The order of the test is defined by alphabetic order, the DB is not necessary reseted between call because the DB is set with the setUpTestData() which is called only once before starting the test of the class. For such test case we should use setUp() method to create the data in DB. This way the DB is reseted before each test.